openshift/settings.demo.sh

```diff
@@ -0,0 +1,19 @@
+export PROJECT_NAMESPACE="devex-von-image"
```

Comment:
Please add a Description to the top of the file to better describe and identify the profile; example, https://github.com/bcgov/essential-services-delivery/blob/master/openshift/settings.business-registration.sh#L1